modules/rosa-quick-start-create-cluster-admin-user.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * rosa_quick_start/rosa-quick-start.adoc
+
+[id="rosa-quick-start-create-cluster-admin-user_{context}"]
+= Creating a cluster admin user for quick cluster access
+
+Before configuring an identity provider, you can create a user with `cluster-admin` privileges for immediate access to your {product-title} (ROSA) cluster.
+
+[NOTE]
+====
+The cluster admin user is useful when you need quick access to a newly deployed cluster. However, Red Hat recommends that you configure an identity provider and grant cluster administrator privileges to the identity provider users as required. For more information about setting up an identity provider for your ROSA cluster, see _Configuring an identity provider and granting cluster access_.
+====
+
+.Prerequisites
+
+* You created an AWS account.
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a ROSA cluster.
+
+.Procedure
+
+. Create a cluster admin user:
++
+[source,terminal]
+----
+$ rosa create admin --cluster=<cluster-name> <1>
+----
+<1> Replace `<cluster_admin>` with the name of your cluster.
```

Comment:
I think this should be "<cluster-name>".

Reply:
@0kashi thank you for your review and for highlighting this. I have subsequently resolved this.